### PR TITLE
update 1.18 source deps

### PR DESCRIPTION
**Pull request by tbrisker**

### CODE REVIEW

**Comment by tbrisker** [comment]
cc/ @xprazak2

File: _includes/manuals/1.18/3.4_install_from_source.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ Foreman uses Bundler to install dependencies and get up and running. This is the
 
 Foreman will run with the following requirements (aside from rubygem dependencies):
 
-* Ruby 2.0, 2.1, 2.2, 2.3 or 2.4
+* Ruby 2.3, 2.4, or 2.5
```
Comment (ekohl) [inline _includes/manuals/1.18/3.4_install_from_source.md]: Do we support 2.5? @mmoll?

File: _includes/manuals/1.18/3.4_install_from_source.md [hunk 2]
```diff
@@ -4,7 +4,7 @@ Foreman uses Bundler to install dependencies and get up and running. This is the
...
 * NodeJS 4.0 or newer
```
Comment (ekohl) [inline _includes/manuals/1.18/3.4_install_from_source.md]: I'm not sure about this. In Travis we only test with node 6 or higher. EPEL also has this version.

File: _includes/manuals/1.18/3.4_install_from_source.md (hunk 1 above)
Comment (mmoll) [reply _includes/manuals/1.18/3.4_install_from_source.md]: No, we don't with 1.18.

File: _includes/manuals/1.18/3.4_install_from_source.md (hunk 2 above)
Comment (mmoll) [reply _includes/manuals/1.18/3.4_install_from_source.md]: Yeah, we should to raise this to 6.14 (the version currently in EPEL).

**Comment by tbrisker** [comment]
thanks @ekohl @mmoll, updated

**Review by ekohl — APPROVED** [review]

**Comment by mmoll** [comment]
merged, thanks @tbrisker!

**Pull request merged**